tests/sentry/api/endpoints/test_team_projects.py

```diff
@@ -96,6 +97,18 @@ def test_invalid_platform(self):
         )
         assert response.data["platform"][0] == "Invalid platform"
 
+    def test_invalid_name(self):
+
+        invalid_name = list(RESERVED_PROJECT_SLUGS)[0]
```

Comment:
You could make this a for loop.

Reply:
Yeah, I didn't know if there's a precedent or preference for that because the two other tests I could find only used the first index/member for their test. I guess if the list gets super long the test will take a while, but idk if it's a big concern ?